.github/workflows/blogs-indexing.yaml

```diff
@@ -0,0 +1,36 @@
+name: Indexing blogs
+on:
+  workflow_dispatch:
+  push:
+    branches: ['search-blogs-ci-indexing']
+    paths:
+      - 'content/blog/**'
+jobs:
+  build-and-indexing-blogs:
+    name: Build and indexing blogs
+    runs-on: ubuntu-latest
+    env:
+      GATSBY_ALGOLIA_APP_ID: ${{ secrets.GATSBY_ALGOLIA_APP_ID }}
+      ALGOLIA_ADMIN_KEY: ${{ secrets.ALGOLIA_ADMIN_KEY }}
+      GATSBY_ALGOLIA_INDEX_NAME: ${{ secrets.GATSBY_ALGOLIA_INDEX_NAME }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node.js and use yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+          cache: 'yarn'
+      - name: Gatsby Cache Folder
+        uses: actions/cache@v2
+        with:
+          key: gatsby-cache-folder
+          path: .cache
```

Comment:
Maybe I'm misreading the readme, but can't [actions/cache](https://github.com/actions/cache#whats-new) cache multiple directories? If it can, should we cache the `.cache` folder and `.public` folder in the same step?

Reply:
Yeah, It looks like that could be done.